third_party/pcsc-lite/naclport/server_clients_management/src/permissions_checking/policy-or-prompting-permissions-checker-unittest.js

```diff
@@ -58,8 +57,8 @@ function FakeTrustedClientsRegistry() {}
 /** @override */
 FakeTrustedClientsRegistry.prototype.getByOrigin = function(origin) {
   if (origin === CLIENT_ORIGIN)
-    return goog.Promise.resolve(TRUSTED_CLIENT_INFO);
```

Comment:
Note: The PR is unfortunately a bit sizy because of mechanical changes like these - switching from Closure Library's (deprecated) Promises to the standard Ecmascript ones.

This is a viral effect of switching to `fetch` which is standard-promise-based and required the code chunks around it to use the same.

Reply:
Yes, eventually I'm planning to switch everything to standard Promises. The current PR is a bit special in that doing both changes simulatenously was much simpler (because the `xhrio` interface used legacy promises), but generally it should be doable gradually.